model/riscv_vmem_pte.sail

```diff
@@ -19,13 +19,20 @@
 
 type pte_flags_bits = bits(8)
 
-// For PTW extensions (non-standard)
-type extPte = bits(64)
+// For PTW extensions (standard)
+type extPte = bits(10)
+
+bitfield PTE_Ext_Flags : extPte = {
+  N    : 9,
+  PBMT : 8 .. 7,
+  RSVD : 6 .. 0
```

Comment:
Minor but can we style this like `reserved` so it doesn't look like it is an actual field?

Reply:
updated.